src/bci_build/package/golang/README.md.j2

```diff
@@ -4,19 +4,36 @@
 
 ## Description
 
-[Go](https://go.dev/) (a.k.a., Golang) is a statically-typed programming language, with syntax loosely derived from C. Go offers additional features such as garbage collection, type safety, certain dynamic-typing capabilities, additional built-in types (for example, variable-length arrays and key-value maps) as well as a large standard library.
+[Go](https://go.dev/) (a.k.a., Golang) is a statically-typed programming
+language, with syntax loosely derived from C. Go offers additional features
+such as garbage collection, type safety, certain dynamic-typing capabilities,
+additional built-in types (for example, variable-length arrays and key-value
+maps) as well as a large standard library.
+
+{%- if image.version.endswith('-openssl') %}
+
+## FIPS 140-2
+
+The image includes a FIPS 140-2 enabled Go wrapper that prefers using OpenSSL
```

Comment:
prefers? I believe it uses it whether you like it or not.

> Go wrapper that uses OpenSSL

Reply:
in my experience it only uses them when dlopen() succeeds, otherwise it falls back to the go stdlib implementation (which is not FIPS certified)